docs/servicenow.itsm.change_request_module.rst

```diff
@@ -46,8 +46,26 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>The group that the change request is assigned to.</div>
+                        <div>The name of the group that the change request is assigned to.</div>
```

Comment:
`docs/*.rst` will be autogenerated at the time of release. I would drop this change from this PR.